src/extension/VSProject.cs

```diff
@@ -382,7 +382,7 @@ private void LoadMSBuildProject()
                     outputPath = commonOutputPath;
 
                 if (outputPath != null)
-                    _configs.Add(name, new ProjectConfig(this, name, outputPath.Replace("$(Configuration)", name), assemblyName));
+                    _configs[name] = new ProjectConfig(this, name, outputPath.Replace("$(Configuration)", name), assemblyName);

```

Comment:
When running this code in my head, it looks like a duplicated section will cause the `outputPath` to reset to the `commonOutputPath` if the duplicated section does not contain an `OutputPath` property.

Reply:
I think, that it's not going to happen. 
`commonOutputPath` is set only when section doesn't have `Condition` attribute.

```
if (name == null)
{
    if (outputPathElement != null)
        commonOutputPath = outputPath;
    continue;
}
```

Reply:
So if you have an unconditional property group defining OutputPath, then the second duplicated section will take that and overwrite the OutputPath defined by the first duplicated section.

```xml
  <PropertyGroup>
    <OutputPath>A</OutputPath>
  </PropertyGroup>
  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
    <OutputPath>B</OutputPath>
  </PropertyGroup>
  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
    <SomethingElse>x</SomethingElse>
  </PropertyGroup>
```

When I mentally follow the code, it looks like it would give me `A` as the output path for the debug configuration.

Reply:
I agree. This looks like it would be sensitive to the order of property groups. We may need to read through the entire file first, saving information, and only make the decisions at the end.

Reply:
True, firstly I misunderstood what You mean.
I've checked VS2017 and it uses path from *last* section with defined `OutputPath`.
If You'd like it, I could try to change the code to do the same, but I think that it needs a lot more changes so You'll have to wait until I'll find some time to do it.

Reply:
I'd be inclined to merge this.

Reply:
OK, so I'll try to update the code..... be patient ;-)

Reply:
@gavardev My comment was suggesting we merge your fix as is for now. I didn't merge it myself because that has to wait for a second team member to OK it.

However, if you prefer to make other changes first, we can re-review when you're ready.

Reply:
Yes, I'd like to make some other changes first.